src/gate.ts

```diff
@@ -156,6 +156,7 @@ export class Gate {
             return
         }
         this.t7.log(`updating ${this.name} state to ${state}`)
+        this.map.shell.stopHourglass()
```

Comment:
As this is clearly part of the UI, I'd rather call it in commands.js and leave the gate to focus on the logic